.github/workflows/ci-tests.patch-external.yml

```
@@ -1,115 +1,139 @@
-# Workflow patches for skipping Google Cloud unit test CI on PRs from external repositories.
-name: Integration Tests on GCP
+# Workflow patches for skipping CI tests on PRs from external repositories
+name: Run tests
 
 # Run on PRs from external repositories, let them pass, and then Mergify will check them.
 # GitHub doesn't support filtering workflows by source branch names, so we have to do it for each
 # job.
 on:
   pull_request:
 
-# IMPORTANT
-#
-# The job names in `ci-integration-tests-gcp.yml`, `ci-integration-tests-gcp.patch.yml` and
-# `ci-integration-tests-gcp.patch-external.yml` must be kept in sync.
+#! IMPORTANT
+#!
+#! The job names in `sub-ci-unit-tests-docker.yml`, `sub-ci-integration-tests-gcp.yml`,
+#! `ci-tests.patch.yml` and `ci-tests.patch-external.yml` must be kept in sync.
 jobs:
-  # We don't patch the testnet job, because testnet isn't required to merge (it's too unstable)
-  get-available-disks:
-    name: Check if cached state disks exist for Mainnet / Check if cached state disks exist
+  build:
+    name: Build CI Docker / Build images
```

Should we change the name to match the job name in `ci-tests.yml`?

```suggestion
    name: Build CI Docker / Build CI image
```